### ISSUE TITLE
Accessibility Report: Lockup Component

### ISSUE THREAD

**Issue by sherakama** [SU-SWS/decanter#321]
Use the AMP tool to scan the [Lockup](http://decanter.stanford.edu/section-components.html#kssref-components-lockup) component for accessibility issues and document them to this ticket.

**Comment by sherakama** [comment]
AMP Report: https://stanford.levelaccess.net/public/reporting/view_report.php?report_id=283425

No violations found.

**Comment by sherakama** [comment]
Chrome Audit: 100%

![Screen Shot 2019-03-13 at 9 00 46 PM](https://user-images.githubusercontent.com/550602/54330479-1b4da480-45d3-11e9-8396-7356e30386d2.png)
